implementations/micrometer-registry-cloudwatch/src/main/java/io/micrometer/cloudwatch/CloudWatchMeterRegistry.java

```diff
@@ -149,6 +150,7 @@ public void onSuccess(PutMetricDataRequest request, PutMetricDataResult result)
     // VisibleForTesting
     class Batch {
         private long wallTime = clock.wallTime();
+        private WarnThenDebugLogger logger = new WarnThenDebugLogger(Batch.class);
```

Comment:
I'd like to move this up to the CloudWatchMeterRegistry class, but I can take care of this in a follow-up polish commit.